docs/architecture/adr-008-dCERT-group.md

```diff
@@ -0,0 +1,129 @@
+Additionally, a special privilege is proposed for the dCERT group:
+the capacity to "circuit-break" (aka. temporarily disable)  a particular
+message path. 
+
+In the future it is foreseeable that the community may wish to expand the roles
+of dCERT with further responsibilities such as the capacity to "pre-approve" a
+security update hard-fork on behalf of the community prior to a full community
+wide vote whereby the sensitive information would be revealed prior to a
+vulnerability being patched on the live network.  
+
+## Decision
+
+The dCERT group is proposed to include an implementation of a `SpecializationGroup`
+as defined in [ADR 007](./adr-007-specialization-groups.md). This will include the 
+implementation of: 
+ - continuous voting
+ - slashing due to breach of soft contract
+ - revoking a member due to breach of soft contract
+ - emergency disband of the entire dCERT group (ex. for colluding maliciously) 
+ - compensation stipend from the community pool or other means decided by
+   governance
+
+This system necessitates the following new parameters: 
+ - blockly stipend allowance per dCERT member (suggested ~$10K/yr of atoms)
+ - maximum number of dCERT members (suggested 7) 
+ - required staked slashable tokens for each dCERT member (suggested $10K of atoms)
+ - unbonding time for dCERT staked token (suggested 3 week)
+ - quorum for suspending a particular member (suggested 5/7) 
+ - proposal wager for disbanding the dCERT group (suggested ~$100000 USD worth of atoms)
+
+These parameters are expected to be implemented through the param keeper such 
+that governance may change them at any given point. 
+
+### Continuous Voting Electionator
+
+An `Electionator` object is to be implemented with continuous voting is to be
+implemented with the following specifications:
+ - All delegation addresses may submit votes at any point which updates their 
+   preferred representation on the dCERT group. 
+ - Preferred representation may be arbitrarily split between address (ex. 50%
+   to John, 25% to Sally, 25% to Carol) 
+ - Addresses which control the greatest amount of preferred-representation are
+   eligible to join the dCERT group (up the _maximum number of dCERT members_)
+ - In order for a new member to be added to the dCERT group they must 
+   send a transaction accepting their admission at which point the validity of
+   their admission is to be confirmed. 
+ - If the dCERT group is already full and new member is admitted, the existing
+   dCERT member with the lowest amount of atoms is kicked from the dCERT group. 
+
+### Staking/Slashing
+
+All members of the dCERT group must stake tokens _specifically_ to maintain
+eligibility as a dCERT member. This staking mechanism must be designed with an
+unbonding period. Slash a particular dCERT member due to soft-contract breach
+should be performed by governance on a per member basis based on the magnitude
+of the breach.  The process flow is anticipated to be that a dCERT member is
```

Comment:
How are they suspended? What if they breach contract but a governance proposal claiming they've done so isn't submitted until right before their unbonding period is over?